src/javascripts/components/mobile-navigation.js

```diff
@@ -63,6 +63,17 @@ MobileNav.prototype.bindUIEvents = function () {
   })
 }
 
+MobileNav.prototype.addHeadings = function () {
+  var $headings = this.$nav.querySelectorAll('.js-app-mobile-nav-subnav__link-heading')
+
+  nodeListForEach($headings, function ($headingText) {
+    const heading = document.createElement('h3')
+    heading.classList = 'app-mobile-nav-subnav__link-heading'
+    $headingText.after(heading)
```

Comment:
According to Can I Use, [`.after`](https://developer.mozilla.org/en-US/docs/Web/API/Element/after) doesn't work in IE11 – although I can't tell for sure because it's erroring on the previous line at the minute.

Unfortunately, there's no polyfill for it in GOV.UK Frontend I don't think we have any prior art for including additional polyfills only in the Design System… 🤔

Reply:
Replaced with `appendChild` [here](https://github.com/alphagov/govuk-design-system/pull/1744/commits/08a03d316eabe5276d42b726dad495d73b3118d0#diff-d9c0c432de139ed8f698b946daa7600de2b353555530aae0cc8717e1ddf7e174R69-R74)

Reply:
Neat solution 👍🏻